### PR TITLE
 Add cross-chain oracle extensions

### PR DESCRIPTION
**Pull request by elv-todd**
PR into develop not master
https://github.com/eluv-io/elv-client-js/pull/179

### CODE REVIEW

File: src/ElvClient.js [hunk 1]
```diff
@@ -637,9 +646,42 @@ class ElvClient {
     ethProvider.pollingInterval = 250;
     this.signer = ethProvider.getSigner();
     this.signer.address = await this.signer.getAddress();
+    await this.SetSignDigestFromWeb3Provider({provider});
```
Comment (elv-kevin) [inline src/ElvClient.js:649]: this should be conditional, shouldn't it?

Comment (elv-todd) [reply src/ElvClient.js:649]: conditional on what?

Comment (elv-kevin) [reply src/ElvClient.js:649]: There's a method called SetSignerFromWeb3Provider that's used for metamask. Is this unrelated to that?

Comment (elv-todd) [reply src/ElvClient.js:649]: ah, yea, this is called from inside SetSignerFromWeb3Provider().  i could have just inlined it there, but, thought maybe someone might want to use it separately.  can rename to SetSignDigest() in the next PR.

**Pull request merged**